src/parameters.toml

```diff
@@ -186,6 +186,60 @@ value = 0.96
 type = "float"
 description = "Emissivity of soil, approximated as context. Reference: CLM5 Documentation, 2020. "
 
+[soil_momentum_roughness_length]
+value = 0.01
+type = "float"
+description = "Source: CLM Tech note 5"
+
+[soil_scalar_roughness_length]
+value = 0.007
+type = "float"
+description = "Source: CLM Tech note 5, using u* of 5m/s"
+
+[snow_momentum_roughness_length]
+value = 0.0024
+type = "float"
+description = "Source: CLM Tech note 5"
+
+[snow_scalar_roughness_length]
+value = 0.08
+type = "float"
+description = "Source: CLM Tech note 5, using u* of 5m/s"
+
+[particle_density_quartz]
+value = 2.66e3
+type = "float"
```

Comment:
Particle density citations:
Quartz: Hillel (1982), De Vries (1966)
Organic matter: Hillel (1982)
Other minerals: Hillel (1982)

Volumetric heat capacity:
Balland and Arp (2005)